sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/EventHandlerDelegateShouldHaveProperArgumentsTest.cs

```diff
@@ -34,5 +34,15 @@ public void EventHandlerDelegateShouldHaveProperArguments()
             Verifier.VerifyAnalyzer(@"TestCases\EventHandlerDelegateShouldHaveProperArguments.cs",
                 new EventHandlerDelegateShouldHaveProperArguments());
         }
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void EventHandlerDelegateShouldHaveProperArgumentsShoudNotStackOverflow()
+        {
+            Verifier.VerifyNoExceptionThrown(@"TestCases\EventHandlerDelegateShouldHaveProperArguments.StackOverflow.Fixed.cs", new []
```

Comment:
We use `.Fixed.cs` convention for code fixes.

Please remove this test method and move the test to `EventHandlerDelegateShouldHaveProperArguments.cs`. Preferably to a separate class/namespace with a comment on what it is testing.